eng/SourceBuild.Version.Details.xml

```diff
@@ -104,9 +104,9 @@
       <Sha>3fb25b8db3bec654e37e71a5b2b7fde14444bc2f</Sha>
       <SourceBuild RepoName="source-build" ManagedOnly="true" />
     </Dependency>
-    <Dependency Name="Microsoft.SourceBuild.Intermediate.source-build-reference-packages" Version="6.0.0-alpha.1.21457.1">
```

Comment:
You should also update the normal Verion.Details.xml.  Going forward, I plan to simply copy the repo's Version.Details.xml into SourceBuild.Version.Details.xml as the starting point for taking new commit shas.

Reply:
OK.  I'll do that on port3 which is finishing up CI in sbrp now.  It'll be out this afternoon.  I like to include packages in groups of about 6 or so.  That way if it causes issues with the tarball build, we can better zero in on the culprit.  Plus, the PRs are easier to manage in SBRP.